objState.rdf

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+
+<!DOCTYPE rdf:RDF [
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#" >
+    <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
+    <!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
+]>
+
+
+<rdf:RDF xmlns="http://fedora.info/definitions/1/0/access/objState"
+     xml:base="http://fedora.info/definitions/1/0/access"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+     <owl:Ontology rdf:about="http://fedora.info/definitions/1/0/access/objState">
+        <rdfs:label xml:lang="en">Fedora Commons Repository - Object Status Property</rdfs:label>
+        <rdfs:comment xml:lang="en">Object status property</rdfs:comment>
+        <owl:versionInfo>v4/2015/03/10</owl:versionInfo>
+    </owl:Ontology>
+
+    <owl:ObjectProperty rdf:about="http://fedora.info/definitions/1/0/access/objState">
+      <rdfs:label xml:lang="en">status</rdfs:label>
+      <rdfs:comment xml:lang="en">Describes the state of a resource, such as active or deleted.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://fedora.info/definitions/v4/repository#Resource"/>
+      <rdfs:range rdf:resource="http://fedora.info/definitions/1/0/access/ResourceStatus"/>
+    </owl:ObjectProperty>
+
+    <owl:Class rdf:about="http://fedora.info/definitions/1/0/access/ResourceStatus">
+      <rdfs:label xml:lang="en">resource state</rdfs:label>
+      <rdfs:comment xml:lang="en">Values of the state property. The default values are active and deleted -- but additional values can be created.</rdfs:comment>
```

Comment:
No, not the way I use that word. `active` would be a default, because it is the choice you get if you make no other choice. That's what a default is, to me. You're talking about the range of choices you have if you add none of your own, and that, to me, sounds more like "standardly-understood" (which sounds awkward) or "community-supported" (hence my suggestion).